scripts/generate_data_for_tests/generate_satellite_test_data.py

```diff
@@ -9,21 +10,46 @@
 import nowcasting_dataset
 from nowcasting_dataset import consts
 
-START = pd.Timestamp("2019-01-01T12:00")
-END = pd.Timestamp("2019-01-01T18:00")
+START = pd.Timestamp("2020-04-01T12:00")
+END = pd.Timestamp("2020-04-01T18:00")
 OUTPUT_PATH = Path(os.path.dirname(nowcasting_dataset.__file__)).parent / "tests" / "data"
 print(OUTPUT_PATH)
 
+# HRV Path
+HRV_SAT_FILENAME = os.path.join(
+    "/mnt/storage_ssd_8tb/data/ocf/solar_pv_nowcasting/nowcasting_dataset_pipeline/satellite/EUMETSAT/SEVIRI_RSS/zarr/v2/hrv_*"
+)
+
+# Non-HRV path
+SAT_FILENAME = os.path.join(
+    "/mnt/storage_ssd_8tb/data/ocf/solar_pv_nowcasting/nowcasting_dataset_pipeline/satellite/EUMETSAT/SEVIRI_RSS/zarr/v2/eumetsat_zarr_*"
+)
+
 
 def generate_satellite_test_data():
     """Main function to make satelllite test data"""
+    output_filename = OUTPUT_PATH / "hrv_sat_data.zarr"
```

Comment:
do you mind adding a few comments in here, just make it supper clear whats going on